OMCompiler/SimulationRuntime/c/util/list.c

```diff
@@ -150,20 +168,43 @@ void *listLastData(LIST *list)
   return list->last->data;
 }
 
-void listPopFront(LIST *list)
+/**
+ * @brief Return first node and pop from list
```

Comment:
Make it clear, that memory for the returned node has to be freed with `freeNode` by the caller of `listPopFrontNode`.

Reply:
I wrote it further down at the `@return` description but I guess nobody looks at that :smiling_face_with_tear:

Reply:
Indeed I did not read it^^. Okay, that's my fault.